### DISCUSSION TITLE
truffle init webpack

### DISCUSSION THREAD

**Discussion by mehrdaad** [ConsenSys-archive/truffle#5996] [Truffle Boxes]
I expect by running "truffle init webpack" I have a webpack template created, but it only inits tuffle in webpack folder.
I tried  "truffle unbox webpack" I got this error:
"Error connecting to https://raw.githubusercontent.com/truffle-box/webpack-box/master/truffle-box.json. Please check your internet connection and try again"

How can I init truffle with webpack template?

**Comment by cipaterson** [comment]
Hi @mehrdaad,

are you still having trouble here?  It works for me, it may have been something temporary.  Please let us know if you are ok now.
